lib/tasks/coverage.rake

```diff
@@ -1,4 +1,6 @@
+# frozen_string_literal: true
+
 task :coverage do
   require 'simplecov'
-  SimpleCov::ResultMerger.merged_result.format!
+  SimpleCov::ResultMerger.merged_result&.format!
```

Comment:
Does it mean that when coverage data is missing, we do nothing? Maybe warn user that no rets were found so nothing is done to avoid confusion as to why nothing happened?

Reply:
I think I'd leave any further coverage fixes/improvements for another PR :upside_down_face: I'll actually open a JIRA to remember, and then we decide whether to fix it, or drop it completely.